.changeset/tiny-spoons-repeat.md

```diff
@@ -0,0 +1,15 @@
+---
+'@backstage/backend-defaults': patch
```

Comment:
Could you split up this changeset with one for `backend-defaults` and one for `plugin-catalog-backend`? only mentioning the changes that apply to each of those pacakges?

Reply:
Done 👍